homeassistant/components/google_assistant/helpers.py

```diff
@@ -161,6 +162,10 @@ def local_sdk_user_id(self):
         """Return the user ID to be used for actions received via the local SDK."""
         raise NotImplementedError
 
+    def get_webhook_id(self, agent_user_id):
+        """Return the webhook ID to be used for actions for a given agent user id via the local SDK."""
+        return self.local_sdk_webhook_id
```

Comment:
Let's get rid of this property altogether. For `cloud/google_config.py` you can just rename the property implementation to `get_webhook_id`.

Reply:
Did this and also created a get_local_agent_user_id to simplify the use of the helper classes (i.e. move the methods back to helper from http).